components/UploadFile.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <form enctype="multipart/form-data" method="post" @submit.prevent>
+    <VueFileAgent
+      ref="vueFileAgent"
+      :theme="'grid'"
+      :multiple="true"
+      :deletable="true"
+      :meta="true"
+      :accept="'image/*,.zip'"
+      :maxSize="'10MB'"
+      :maxFiles="14"
+      :helpText="'Choose images or zip files'"
+      :errorText="{
+        type: 'Invalid file type. Only images or zip Allowed',
+        size: 'Files should not exceed 10MB in size',
+      }"
+      @select="filesSelected($event)"
+      @beforedelete="onBeforeDelete($event)"
+      @delete="fileDeleted($event)"
+      v-model="fileRecords"
+      name="files"
+    ></VueFileAgent>
+    <button :disabled="!fileRecordsForUpload.length" @click="save()">
+      Upload {{ fileRecordsForUpload.length }} files
+    </button>
+  </form>
+</template>
+<script>
+import axios from '@nuxtjs/axios'
```

Comment:
@lemuelapa you do not have to declare this one here.. declare it on the nuxt.config.js